icubam/backoffice/handlers/users.py

```diff
@@ -39,50 +40,111 @@ def get(self):
       '{}?id={}'.format(UserHandler.ROUTE, self.user.user_id))
 
 
-
 class UserHandler(base.BaseHandler):
   ROUTE = "/user"
 
+  """Creates ICU DTOs based on ALL ICUs and those from the subset."""
+  def prepare_icus(self, subset_icus_ids):
```

Comment:
maybe call those selected_icus instead of subset ?

Reply:
sure, also renamed the methods to be more like "select_" instead of "prepare_"